components/cdc/src/endpoint.rs

```diff
@@ -48,7 +48,6 @@ pub enum Task {
         downstream_id: DownstreamID,
         entries: EntryBatch,
     },
-    #[cfg(not(validate))]
```

Comment:
Why is this removed?

Reply:
It's actually wrong, it should be `#[cfg(not(feature = "validate")]`, so I think it's better to be removed to reduce mistakes and complexity.